src/tag/examples.js

```diff
@@ -59,12 +70,18 @@ export default {
             <Tag
               key={color}
               color={color}
+              kind="custom"
```

Comment:
Is it possible to avoid an explicit `custom` kind?

Reply:
I could remove the `custom` kind, but I felt it was confusing to leave `color` and allow `kind` to be set. I wanted a more explicit mechanism for enabling the color prop.

Reply:
I see, sounds good